indykite/common_schema.go

```diff
@@ -57,6 +59,29 @@ func regionSchema() *schema.Schema {
 	}
 }
 
+func ikgSizeSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: `IKG size that will be allocated, which correspond also to amount of CPU nodes (default 2GB).`,
```

Comment:
what does this mean? That 2GB == 2 CPU nodes? and 512GB == 512 CPU nodes?

Reply:
Either way, there are some grammar mistakes:
should be "correspond**s**" and "to **the** **number** of CPU nodes..."